pkg/primaza/workercluster/agentsvc.go

```diff
@@ -69,12 +69,7 @@ func createAgentSvcDeployment(ctx context.Context, cli *kubernetes.Clientset, na
 	}
 
 	dep := obj.(*appsv1.Deployment)
-	r, err := cli.AppsV1().Deployments(namespace).Create(ctx, dep, metav1.CreateOptions{})
-	if err != nil {
-		return nil, fmt.Errorf("error creating deployment: %w", err)
-	}
-
-	return r, nil
+	return dep, nil
```

Comment:
this does not sound correct to me, is it done on purpose?

Reply:
Following today's discussion, this has been reverted.